source/MissionAction.cpp

```diff
@@ -33,7 +33,15 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 using namespace std;
 
 namespace {
-	void DoGift(PlayerInfo &player, const Outfit *outfit, int count, UI *ui)
+	void DoGiftShip(PlayerInfo &player, const Ship *model, const std::string &name, UI *ui)
+	{
+		player.GiftShip(modelName, name);
```

Comment:
You also need to rename the variable here.
```suggestion
		player.GiftShip(model, name);
```